src/core/template-host.ts

```diff
@@ -83,21 +84,18 @@ class TemplateHost {
         };
     }
 
-    public addNested(props: {
-        name: string;
-        render: any;
-        component: any;
-    }): void {
+    public addNested(props: ITemplateProps): void {
         const name: string = props.name;
         this._nestedTemplateProps[name] = {
             component: props.component,
-            render: props.render
+            render: props.render,
+            children: () => props.children
         };
         const propsGetter = (prop) => this._nestedTemplateProps[name][prop];
 
         const templateDescr: ITemplateDescr = {
             name,
-            propName: !!props.component ? "component" : "render",
+            propName: !!props.component ? "component" : !!props.render ? "render" : "children",
```

Comment:
Isn't it possible to drop `propName` and modify `propsGetter` this way:
```ts
 const propName = !!props.component ? "component" : !!props.render ? "render" : "children";
 const propsGetter = () => this._nestedTemplateProps[name][propName];
```

Reply:
Btw, why we have to pass a getter, not a property?

I mean, could we switch to
```ts
const prop = props.component || props.render || props.children
```